samples/bluetooth/peripheral_nucleo_wb55rg/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _bluetooth-peripheral_nucleo_wb55rg:
+
+Bluetooth: STM32WB55 BLE peripheral demo
+########################################
+
+Overview
+********
+
+This application demonstrates BLE peripherals on an `ST Nucleo WB55RG`_
+by exposing a battery service and vendor-specific GATT services.
+
+Requirements
+************
+
+* `ST BLE Sensor Android app <ST BLE Sensor app>`_
+* A board with STM32WB55 like ST Nucleo WB55RG
+* :ref:`Zephyr kernel environment configured for ST Nucleo WB55RG <nucleo_wb55rg_board>`
+
+Building and Running
+********************
+This sample can be found under :zephyr_file:`samples/bluetooth/peripheral_nucleo_wb55rg` in the
+Zephyr tree.
+
+west build -b nucleo_wb55rg samples/bluetooth/peripheral_nucleo_wb55rg
```

Comment:
Sorry, didn't comment on this earlier.  These build/flash commands should be written as:
```
.. zephyr-app-commands::
   :zephyr-app: samples/bluetooth/peripheral_nucleo_wb55rg
   :board: nucleo_wb55rg
   :goals: build flash
   :compact:

```

Reply:
done